src/routes/Quote.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+// import QuoteGenerator from '../components/Quote';
```

Comment:
**I can see you put great efford for creating such website. I really like it.**

- [ ] However, we must keep our project as clean as possible. At that point, I can see that you have a commented code in your project. If this code is not have any necessary action, please remove it. Thank you!